### PR TITLE
completed javascript-loops-practice - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-03-23 at 7 57 57 PM
Screenshot 2023-03-23 at 7 57 47 PM
Screenshot 2023-03-23 at 7 58 03 PM

### CODE REVIEW

File: javascript-loops-practice/countdown.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* exported countdown */
+function countdown(number) {
+  const numbers = [];
+  let currentNumber = number;
+  while (currentNumber >= 0) {
+    console.log(currentNumber);
```
Comment (shawnkost) [inline javascript-loops-practice/countdown.js]: Delete any unused console logs! These are purely for debugging / testing purposes.

File: javascript-loops-practice/odd-or-even.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+// even using strings 'odd' 'even'
+// will use a for loop to run through the input and will check for odd or even
+// numbers
+// pretty sure will need %2 === 0
+function oddOrEven(numbers) {
+  const output = [];
+  for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+      output.push('even');
+    } else output.push('odd');
```
Comment (shawnkost) [inline javascript-loops-practice/odd-or-even.js]: You should have another `{` after `else`. 

```
} else {
```

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple small changes requested here Bryan!

File: javascript-loops-practice/odd-or-even.js (hunk 2 above)
Comment (bbrown949) [reply javascript-loops-practice/odd-or-even.js]: i tried that and it broke the code...

I removed the console log and and will push again for you to take a look

Comment (shawnkost) [reply javascript-loops-practice/odd-or-even.js]: Did you add another bracket after the else statement? The whole code should look like this
```
function oddOrEven(numbers) {
  const output = [];
  for (let i = 0; i < numbers.length; i++) {
    if (numbers[i] % 2 === 0) {
      output.push('even');
    } else {
      output.push('odd');
    }
  } 
  return output;
}
```

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change requested! I added a comment to our previous comment thread

**Comment by bbrown949** [comment]
fixed 👍🏼

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**